service/evidence/evidence_store.go

```diff
@@ -219,8 +223,12 @@ func (svc *Service) ListEvidences(ctx context.Context, req *evidence.ListEvidenc
 
 	// In any case, we need to make sure that we only select evidences of cloud services that we have access to
 	if !all {
+
 		query = append(query, "cloud_service_id IN ?")
-		conds = append(conds, allowed)
+
+		// Add all cloud_service_ids to the slice
+		ids = util.StringToAnySlice(allowed)
```

Comment:
are you sure about this? I thought we had several places where we are using it like before and I thought it was working fine?

It is also used like this in https://gorm.io/docs/query.html#String-Conditions, with just a regular array.

Reply:
I guess it's never really worked before. It's just not noticed because we never had a list of cloud_service_ids before. 
The problem is the following: Before we got a list that looked like this '(("11111111-1111-1111-1111-111111111111","22222222-2222-2222-2222-222222222222"))' , but we need a list that looks like that '(("11111111-1111-1111-1111-111111111111"),("22222222-2222-2222-2222-222222222222"))'. Maybe it is also possible to change that in the List() function, I don't know.

Reply:
I added a new test with 2 cloud_service_ids. You can try it out by changing the lines 228-231 back.

Reply:
I changed the lines back to
```
query = append(query, "cloud_service_id IN ?")
conds = append(conds, allowed)
```

and your test succeeds without a problem 🤷